### DISCUSSION TITLE
Help. Transfer went through but ETH Shows up as Bridge ETH

### DISCUSSION THREAD

**Discussion by anonemouseott** [wormhole-foundation/wormhole#431] [Problems / bugs] [answered]
So the Transaction passed through Metamask to SOL successfully, but SOL dApps do not recognize the ETH that's in my Phantom wallet. So I checked and the ETH shows up as Wrapped ETH (Wormhole bridge) and in the list of transfers there's a failed transfer with an error. it takes me to that explorer link: 
https://explorer.solana.com/tx/39MyvcyY2xJMC6pkW6iAS39UonphaCmhSBRkvUP6vMLsvH5K7gR8jixJfjt8t536Up1egFEX5UViTHnUEDWT9CXi
Here's the transaction history for the same token: https://explorer.solana.com/address/6aPC6AX35W4homAHFZifNWURojPTc3FMYL2wTWSa61tg

What's the next step forward? Any help would be greatly appreciated!

**Comment by leoluk** [comment, marked as answer]
You have Wormhole-wrapped wEth now. You either need to find a pool or dApp that supports this token, or send it back to Eth.

**Reply by anonemouseott** [reply]
Shouldn't the bridge automatically unwrap it to regular wETH?

**Reply by leoluk** [reply]
There is no "regular wETH" - there's just different varieties of bridged wETH.

**Reply by anonemouseott** [reply]
Ok thanks, you can mark it as solved

**Reply by leoluk** [reply]
Sure, and if you have any other questions- happy to help!
